src/main/scala/util/gen.scala

```diff
@@ -14,15 +14,15 @@ object gen {
     val dpia_e = toDPIA(e)
     val p = shine.C.ProgramGenerator.makeCode(dpia_e, name)
     SyntaxChecker(p.code)
-    println(p.code)
+    //println(p.code)
     p
   }
 
   def OpenMPProgram(e: rise.core.Expr, name: String = "foo"): shine.OpenMP.Program = {
     val dpia_e = toDPIA(e)
     val p = shine.OpenMP.ProgramGenerator.makeCode(dpia_e, name)
     SyntaxChecker(p.code)
-    println(p.code)
+    //println(p.code)
```

Comment:
we can remove the comment?

Reply:
I have kept the `println` uniformly for all versions.